Floofbot/Migrations/20200502011652_InitialCreate.cs

```diff
@@ -22,31 +36,33 @@ protected override void Up(MigrationBuilder migrationBuilder)
                     UserKickedChannel = table.Column<ulong>(nullable: false),
                     UserMutedChannel = table.Column<ulong>(nullable: false),
                     UserUnmutedChannel = table.Column<ulong>(nullable: false),
-                    IsOn = table.Column<bool>(type: "INT", nullable: false)
+                    IsOn = table.Column<bool>(nullable: false)
```

Comment:
Do we want the other tables affected by this migration?
I think you can manually edit this file so that the only change is the creation of the AdminConfig table.

Reply:
I agree, the auto generated code is kind of funky here.